### DISCUSSION TITLE
Had to change IP of manager

### DISCUSSION THREAD

**Discussion by BSUNetSec** [Security-Onion-Solutions/securityonion#12772] [2.4] [answered]
### Version

2.4.60

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

80

### RAM

754G

### Storage for /

1TB

### Storage for /nsm

35TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Greetings,

So due to issues beyond my control, I am forced to change management IP's for my manager and all my sensors.  I have changed the IP on the manager but I am guessing that it is hard coded into a large amount of parts.  Since it is a distributed install, the so-ip-update doesn't appear to work.  Is there a way to update what I need or is this nuke from orbit and start over kind of thing.

Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Acewiza** [comment]
I ran that on a standalone just a few weeks ago after a big network hardware swap-out/re-org and it worked fine.

So the Manager is OK now?  New IP on him?

**Reply by BSUNetSec** [reply]
Not really. Got all of the services to come back up after updating IP in several config files in /opt/so/ up but not able to ssh into it or get the web interface to come up at all.  not sure if I need to update something for the firewall but not able to find anything that would be blocking it.

**Comment by Acewiza** [comment]
No telling from me but, without reverting from those pesky issues, un-skrewing that is gonna be non-trivial.

**Reply by BSUNetSec** [reply, marked as answer]
Ended up just flattening the manager, starting over and adding all my sensors back to the manager
